uint64 timestamp
uint64 sequence_num
uint16 module_name
uint16 status
uint64 is_debag
uint64 measured_timestamp
uint8[3] version
uint64 token
uint64 token_timestamp
string detail

